test/test_config.py

```diff
@@ -0,0 +1,31 @@
+"""Tests related to ansiblelint.config module."""
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+
+def test_call_from_outside_venv() -> None:
+    """Asserts ability to be called without venv activation."""
+    py_path = os.path.dirname(sys.executable)
+    # first validate that normally we do not inject the path
+    proc = subprocess.run(
+        f"{py_path}/ansible-lint --version",
+        shell=True,
+        check=True,
+        capture_output=True,
+        universal_newlines=True,
+    )
+    assert "PATH altered to include" not in proc.stderr
+
+    # env -i is used in order to prevent the process from inheriting PATH or
+    # other environment variables from the current process, so we emulate being
+    # called from outside the venv.
+    proc = subprocess.run(
+        f'env -i HOME="{Path.home()}" {py_path}/ansible-lint --version',
+        shell=True,
```

Comment:
I'd just use the `env` arg for this instead of requiring shell mode.
```suggestion
        (f"{py_path}/ansible-lint", "--version"),
        env={
            "HOME": Path.home(),
        },
```